src/classes/EntitySqlBuilder.php

```diff
@@ -170,14 +170,16 @@ private function usersTeams(): void
         $this->selectSql[] = "users.firstname,
             users.lastname,
             users.orcid,
-            CONCAT(users.firstname, ' ', users.lastname) AS fullname";
+            CONCAT(users.firstname, ' ', users.lastname) AS fullname,
+            teams.name AS team_name";
 
         $this->joinsSql[] = 'LEFT JOIN users
             ON (users.userid = entity.userid)';
         $this->joinsSql[] = sprintf(
             'LEFT JOIN users2teams
                 ON (users2teams.users_id = users.userid
-                    AND users2teams.teams_id = %s)',
+                    AND users2teams.teams_id = %d)
+            LEFT JOIN teams ON (entity.team = teams.id)',
```

Comment:
Won't this limit the results to only the team, even if the can filter adds additional teams? The additional teams would need their own join, right?

Reply:
Seems to be working fine.